spec/loophole-spec.coffee

```diff
@@ -28,7 +28,47 @@ describe "Loophole", ->
       result = allowUnsafeNewFunction -> 42
       expect(result).toBe 42
 
+    it "works with async callback", ->
+      result = null
+
+      runs ->
+        allowUnsafeNewFunction (callback) ->
+          setTimeout(
+            ->
+              result = 42
+              callback()
+            100
+          )
+
+      waitsFor(
```

Comment:
You can pass them in any order in my experience and we typically do the name and timeout first.

Hear you on Jasmine 2.0, but that would be another PR. I'm willing to limp along with 1.3 for a bit longer.